### DISCUSSION TITLE
Error while pushing the files to git

### DISCUSSION THREAD

**Discussion by Nambunga** [IBAHCM/RPiR#279] [Archive] [answered]
@richardreeve @soniamitchell HI! while trying to push my files into git, I encountered the error as seen in the attachment. Could you please help me with this?

![IMG_4989](https://user-images.githubusercontent.com/77630539/107645662-de800180-6c70-11eb-9a1c-946a367d5ea1.jpeg)

**Comment by soniamitchell** [comment, marked as answer]
Hi @Nambunga, GitHub is complaining that your files are too large. Don't forget to add these file names to your .gitignore file (so that they're ignored by git) before committing. You can do that, like this:

>```
>  usethis::edit_git_ignore("project")
> ```
> And add the following lines to the end of the .gitignore file that is opened:
>
> ```
> ViewTax.txt
> ViewFullTable.zip
> ViewFullTable
> TSMAttributes.txt
> ViewFullTable.txt
> ViewFullTable.pdf
> ```

These files should then disappear from your git pane (you might have to click refresh).

**Reply by richardreeve** [reply]
@Nambunga did you ever fix this?

**Reply by Nambunga** [reply]
@richardreeve Yes, this was fixed perfectly.
